tests/foreman/cli/test_contenthost.py

```diff
@@ -401,6 +407,7 @@ class TestCHKatelloAgent(CLITestCase):
     activation_key = None
 
     @classmethod
+    @skip_if_not_set('clients')
```

Comment:
Can you please run the tests and paste test results here? In the past we had trouble skipping the class and we had to skip individual test functions.